composer.json

```diff
@@ -32,7 +32,7 @@
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
         "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/symfony": "^2.8.7, !=2.8.10 || ^3.0",
```

Comment:
Hm... On the second thought it might bring a confusion that sf3 is already supported (let's imagine some dev briefly looked on deps). If that makes sence version constraint for symfony could be left untouched.